tensorflow/tools/ci_build/Dockerfile.micro

```diff
@@ -1,7 +1,7 @@
 # Use a prebuilt Python image instead of base Ubuntu to speed up the build process,
 # since it has all the build dependencies we need for Micro and downloads much faster
 # than the install process.
-FROM python:3.9.0-buster
+FROM python:3.12.0a3-slim-bullseye
```

Comment:
Can we use a stable release instead of an alpha version? That is, 3.11, instead of 3.12.0a3?

Reply:
Ping on this, @BlackPeter13

Reply:
Haven't tested it with 3.11 but I think it won't solve all the vulnerabilityes that are solved with the proposed version.